client/rest-high-level/src/main/java/org/elasticsearch/client/RestHighLevelClient.java

```diff
@@ -949,6 +949,11 @@ public final void fieldCapsAsync(FieldCapabilitiesRequest fieldCapabilitiesReque
             FieldCapabilitiesResponse::fromXContent, listener, emptySet());
     }
 
+    /**
+     * @deprecated If creating a new HLRC ReST API call, consider creating new actions instead of reusing server actions. The Validation
+     * layer has been added to the ReST client, and requests should extend {@link Validatable} instead of `ActionRequest`.
```

Comment:
Why backticks in the Javadoc instead of using `{@link}` or `{@code}`? I would tend to the user former here, it will resolve as long as `ActionRequest` is on the class `path and then when we reach that glorious day when it isn't, our Javadocs will fail to build and we will know it's time to remove this Javadoc!